README.md

```diff
@@ -563,7 +563,36 @@ The universal object is where you define all of the custom metadata associated w
 
 **expirationDate**: The date when the content will not longer be available or valid. Currently, this is only used for Spotlight indexing but will be used by Branch in the future.
 
+
+### Tracking User Actions and Events
+Use `BranchEvent` class to track special user actions or application specific events beyond app installs, opens, and sharing . You can track events such as when a user adds an item to an on-line shopping cart, or searches for a keyword etc.
+`BranchEvent` provides interface to add contents represented by BranchUniversalObject in order to associate contents with events.
+Analytics about your app's BranchEvents can be found on the Branch dashboard, and BranchEvents also provide tight integration with many third party analytics providers.
```

Comment:
Sentence is too long. Maybe split it up into two sentences. 

You can view analytics for the BranchEvents you fire on the Branch dashboard. BranchEvents provide a seamless integration with many third party analytics providers like <X,Y>
@Sarkar  what should we say X,Y are?

Reply:
Google Analytics, Criteo